internal/cmd/skupper/debug/nonkube/debug.go

```diff
@@ -21,13 +44,372 @@ func NewCmdDebug() *CmdDebug {
+			}
+		}
+	}
+
+	certificates, err := cmd.certificateHandler.List()
+	if err == nil && certificates != nil && len(certificates) != 0 {
+		for _, certificate := range certificates {
+			err := utils.WriteObject(certificate, path+"Certificate-"+certificate.Name, tw)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	secrets, err = cmd.secretHandler.List(opts)
+	if err == nil && secrets != nil && len(secrets) != 0 {
+		for _, secret := range secrets {
+			err := utils.WriteObject(secret, path+"Secret-"+secret.Name, tw)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	configMaps, err := cmd.configMapHandler.List()
+	if err == nil && configMaps != nil && len(configMaps) != 0 {
+		for _, configMap := range configMaps {
+			err := utils.WriteObject(configMap, path+"ConfigMap-"+configMap.Name, tw)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	//runtime/router
+	path = rpath + "router/"
+	skrPath := api.GetInternalOutputPath(cmd.namespace, api.RouterConfigPath+"/skrouterd.json")
+	//skrPath := pathProvider.GetRuntimeNamespace()+ api.
+	skrouterd, err := os.ReadFile(skrPath)
+	if err == nil && skrouterd != nil {
+		utils.WriteTar(path+"skrouterd.json", skrouterd, time.Now(), tw)
+	}
+
+	//logs and skupper-router statistics
+	if err := os.Setenv("SKUPPER_PLATFORM", platform); err == nil {
+		cli, err := internalclient.NewCompatClient(os.Getenv("CONTAINER_ENDPOINT"), "")
+		if err == nil {
+			rtrContainerName := cmd.namespace + "-skupper-router"
+			if container, err := cli.ContainerInspect(rtrContainerName); err == nil {
+				encodedOutput, _ := utils.Encode("yaml", container)
+				utils.WriteTar(rpath+"Container-"+container.Name+".yaml", []byte(encodedOutput), time.Now(), tw)
+			}
+
+			localRouterAddress, err := runtime.GetLocalRouterAddress(cmd.namespace)
+			if err == nil {
+				for x := range flags {
+					skStatCommand := []string{
+						"/bin/skstat", flags[x],
+						"-b", localRouterAddress,
+						"--ssl-certificate", "/etc/skupper-router/runtime/certs/skupper-local-client/tls.crt",
+						"--ssl-key", "/etc/skupper-router/runtime/certs/skupper-local-client/tls.key",
+						"--ssl-trustfile", "/etc/skupper-router/runtime/certs/skupper-local-client/ca.crt",
+					}
+
+					out, err := cli.ContainerExec(rtrContainerName, skStatCommand) //strings.Split(skStatCommand, " "))
+					if err == nil {
+						utils.WriteTar(path+"skstat/"+rtrContainerName+"-skstat"+flags[x]+".txt", []byte(out), time.Now(), tw)
+					}
+				}
+			}
+
+			logs, err := cli.ContainerLogs(rtrContainerName)
+			if err == nil {
+				utils.WriteTar(rpath+"logs/"+rtrContainerName+".txt", []byte(logs), time.Now(), tw)
+			}
+
+			ctlContainerName := "system-controller"
+			if container, err := cli.ContainerInspect(ctlContainerName); err == nil {
+				encodedOutput, _ := utils.Encode("yaml", container)
+				utils.WriteTar(rpath+"Container-"+container.Name+".yaml", []byte(encodedOutput), time.Now(), tw)
+			}
+
+			logs, err = cli.ContainerLogs(ctlContainerName)
```

Comment:
If using the `linux` platform, we need to capture logs too.
In that case, you could run something like:

In rootful mode:
```
journalctl -u skupper-<namespace>.service --no-pager --all
```

In rootless mode:
```
journalctl --user -u skupper-<namespace>.service --no-pager --all
```

Reply:
added logs for linux sites